tests/shaders/reflection/Semantics.hlsl

```diff
@@ -0,0 +1,14 @@
+﻿float4 ReflectionMain1(in float3 pos : POSITION, in float2 uv : TEXCOORD) : SV_Position
+{
+    return float4(pos.xxx, uv.x);
+}
+
+float4 ReflectionMain2(in float3 pos : POSITION, in float2 uv : TEXCOORD, in uint vertexId : SV_VertexID) : SV_Position
```

Comment:
I would add SV_Barycentrics, SV_InstanceID and any others you can think of / find